Makefile

```diff
@@ -276,9 +275,11 @@ test: $(ROM)
 #### Various Recipes ####
 
 $(ROM): $(ELF)
-	$(ELF2ROM) -cic 6105 $< $@
+	$(OBJCOPY) -O binary $< $@
+	python3 tools/pad_rom.py $@
```

Comment:
I'm not thrilled with using python for any build step, even if this is probably already faster overall than the previous elf2rom solution. I don't think it's a disaster if the checksum tool also did the padding, but if that's no good surely this is possible with a bash script?

Reply:
btw sure we can not use python, would a C utility be fine

Reply:
Is there really any difference? Does it matter the value used to pad the rom? Do the rom need to be padded?

tbh I wouldn't be against to just pad the rom with zeroes instead of using two different values.
For diffing against the baserom we could just modify/strip the padding. We already mess with the baserom in `fixbaserom.py`, so I think we could consider it

Reply:
Actually, taking a look at `fixbaserom.py` seems like _we_ are the ones that are padding the ROM with `0xFF`.
https://github.com/zeldaret/oot/blob/0ce03133425867723dd81fc66f57bbb8d40aaf7a/fixbaserom.py#L76-L77
Changing that to pad using `0` instead could allow us to simply use objcopy to pad the rom:
https://github.com/Dragorn421/oot/compare/meta_buildsystem_cleanup...AngheloAlf:oot:meta_buildsystem_cleanup_padding
(I had to change the hash because the changes to the rom makes it change obviously )

Reply:
The explicit 0xFF padding is somewhat recent: https://github.com/zeldaret/oot/pull/987

Reply:
taken to discord https://discord.com/channels/688807550715560050/688851317593997489/1127217091397812327
will try to reach experts